build/cli.js

```diff
@@ -75,6 +77,14 @@ async function buildDocuments() {
       fs.writeFileSync(liveSamplePath, html);
     }
 
+    for (const filePath of fileAttachments) {
+      // TODO: Consider using symlinks instead. At least as an option.
+      // The assumption is that writing a symlink is much faster than
+      // cloning the image to another location on disk.
```

Comment:
It's not true. I've checked. We don't have that many images to make it not worth it. I need to update the TODO comment.